lib/framework/sync_job_runner.rb

```diff
@@ -55,14 +55,19 @@ def cron_parser(cronline)
     end
 
     def should_sync?
-      return false unless @connector_settings.scheduling_settings[:enabled]
-      return true if @connector_settings[:sync_now]
+      # sync_now should have priority over cron
+      return true if @connector_settings[:sync_now] == true
+      scheduling_settings = @connector_settings.scheduling_settings
+      return false unless scheduling_settings[:enabled] == true
 
-      last_synced = @connector_settings.scheduling_settings[:last_synced]
+      last_synced = scheduling_settings[:last_synced]
       return true if last_synced.nil? || last_synced.empty? # first run
 
       last_synced = Time.parse(last_synced) # TODO: unhandled exception
       sync_interval = scheduling_settings['interval']
+      if sync_interval.nil? || sync_interval.empty? # no interval configured
```

Comment:
I strongly feel this place deserves a log info line

Reply:
It clutters the log very much when the idle interval is short and no interval is configured. We might have it as a debug level perhaps?